docs/reference/migration/migrate_2_0.asciidoc

```diff
@@ -408,6 +408,9 @@ In addition some query builders have been removed or renamed:
 * `filtered(...)` removed. Use `filteredQuery(...)` instead.
 * `inQuery(...)` removed.
 
+`TermsLookupQueryBuilder` has been removed, please use `TermsQueryBuilder` with no values and set the corresponding
```

Comment:
ha then I removed, I thought this would be part of refactoring though. Happy to open a PR on master.